addons/account/models/account_move.py

```diff
@@ -2241,6 +2240,10 @@ def is_internal_partner(partner):
         return move
 
     def post(self):
+        if self.env.context.get('soft_post'):
```

Comment:
Because it is not specific to the wizard, everywhere we have a wizard/action that generates (potentially) posted entries in the future, we should use this

Reply:
Is there any value in making it a context parameter, then ? Why not just an optional parameter in post ?

Reply:
Using the context seems indeed not that much useful. 
It can potentially be used to be sure an entire process doesn't post entries in the future, if it is a more complex function that could create entries at many places (xchange difference, and things like that), but I don't see a usecase right now.

I'll use an optional param.